### PR TITLE
Page Titles

### PR DESCRIPTION
**Pull request by Sir3lit**
Adds page titles in the formate of  <server name> | <page> for the end user.

Console
FileManager
Etc.

### CODE REVIEW

**Comment by DaneEveritt** [comment]
Let's use https://github.com/nfl/react-helmet to handle adjusting the title and such, allows components to handle them at whatever layer they want, and deeper components can make modifications easily.

File: resources/scripts/components/server/backups/BackupContainer.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default () => {
 
     const backups = ServerContext.useStoreState(state => state.backups.data);
     const setBackups = ServerContext.useStoreActions(actions => actions.backups.setBackups);
+    const server = ServerContext.useStoreState(state => state.server.data!);
```
Comment (DaneEveritt) [inline resources/scripts/components/server/backups/BackupContainer.tsx:20]: `const { name } = useServer()` is my new method of doing this for the sake of readability and not endlessly typing the same stuff over and over.

I also generally attempt to keep these in some sort of length based order/grouping by what they're doing (but its kinda arbitrary, so don't worry too much).

**Comment by Sir3lit** [comment]
Now sure how react works, kinda just pieced it together.

**Comment by DaneEveritt** [comment]
@Sir3lit did you forget to push it up?

**Comment by Sir3lit** [comment]
I was referring to how it is now, spent a few hours last night trying to move it to react-helmet but with my limited knowledge of react. I wasn't able to get anything working.

**Comment by DaneEveritt** [comment]
Gotcha, I'll find some time Friday/this weekend to get with you and help ya with this.

**Pull request closed without merging**